hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/cluster/ClusteringPlanActionExecutor.java

```diff
@@ -71,7 +71,7 @@ protected Option<HoodieClusteringPlan> createClusteringPlan() {
       return Option.empty();
     }
 
-    if (config.isAsyncClusteringEnabled() && config.getAsyncClusterMaxCommits() > commitsSinceLastClustering) {
+    if (config.scheduleInlineClustering() && config.getAsyncClusterMaxCommits() > commitsSinceLastClustering) {
       LOG.warn("Not scheduling async clustering as only " + commitsSinceLastClustering
           + " commits was found since last clustering " + lastClusteringInstant + ". Waiting for "
```

Comment:
Makes sense ~

Reply:
Maybe you should add another check for inline plan scheduling.

Reply:
done.

Reply:
done.

Reply:
Could you merge this with the `if` branch above, like:
```
if ((config.isAsyncClusteringEnabled() || config.scheduleInlineClustering()) && config.getAsyncClusterMaxCommits() > commitsSinceLastClustering) {
```
? The error message is the same.